src/vmm/src/vstate.rs

```diff
@@ -20,7 +20,12 @@ use arch::aarch64::gic::GICDevice;
 #[cfg(target_arch = "x86_64")]
 use cpuid::{c3, filter_cpuid, t2, VmSpec};
 #[cfg(target_arch = "x86_64")]
-use kvm_bindings::{kvm_pit_config, CpuId, KVM_MAX_CPUID_ENTRIES, KVM_PIT_SPEAKER_DUMMY};
+use kvm_bindings::{
```

Comment:
I kinda disagree with the assessment that it is out of the scope of this PR since this PR adds around 35% of the total number of lines in the file. It is in the scope of this PR, but I do agree with you that it is better to do it after we merge the refactoring branch back to master so we can have a smoother merge.

Reply:
Ok ok line number police :policewoman: I am sorry I was going too mine lines per file. I've filed myself a ticket in #1605

Reply:
:laughing: well, I think we might add more lines, errors and functionality in vstate.rs and it might become unmanagable somewhat similar to what we previously had in vmm/src/lib.rs